.github/workflows/redpanda-build.yml

```diff
@@ -27,8 +27,11 @@ jobs:
             apt-get install -y git sudo pkg-config
 
         - name: install dependencies
-          run: ./install-dependencies.sh
-
+          run: |
+            ./install-dependencies.sh
+            apt-get install -y gcc-10 g++-10
```

Comment:
Nit: `g++-10` requires `gcc-10`, so no need to specify both.

Reply:
it's there, right?